### PR TITLE
Styling/classroom-and-dash (KHO-178)

### PR DESCRIPTION
**Pull request by nandini-ghosh**
# Checklist

- [x] I have performed a self-review of my code
- [x] I have reached out to another developer to review my code
- [x] New and existing unit tests pass locally with my changes
- [x] All Lint and CI checks are passing

### CODE REVIEW

**Review by kennybc — CHANGES_REQUESTED** [review]

File: frontend/src/pages/Classrooms/Select/index.tsx [hunk 1]
```diff
@@ -50,35 +52,49 @@ const ClassroomSelection: React.FC = () => {
...
-            ))}
-            <TableRow className="add-row">
-              <TableCell style={{ textDecoration: "none" }}>
-                <Link to={`/app/classroom/create?org_id=${orgID}`}>
-                  <MdAdd />Create a new classroom
-                </Link>
+              <TableCell>
+                <div className="Selection__tableHeaderText">Classroom Name</div>
+                <div className="Selection__tableHeaderButton">
+                  <Button size="small" onClick={() => navigate(`/app/classroom/create?org_id=${orgID}`)}>
```
[the remark this answers is not in the document]
Comment (nandini-ghosh) [reply frontend/src/pages/Classrooms/Select/index.tsx]: done! thanks

File: frontend/src/pages/Dashboard/index.tsx [hunk 2]
```diff
@@ -94,28 +99,33 @@ const Dashboard: React.FC = () => {
...
+
           <div className="Dashboard__assignmentsWrapper">
-            <h2 style={{ marginBottom: 0 }}>Assignments</h2>
+            <div className="Dashboard__assignmentsHeader">
+              <h2 style={{ marginBottom: 0 }}>Assignments</h2>
+              <div className="Dashboard__createAssignmentButton">
+                <Button
+                  variant="secondary"
+                  size="small"
+                  onClick={() => (navigate(`/app/assignments/create?org_name=${selectedClassroom?.org_name}`))}>
```
[the remark this answers is not in the document]
Comment (nandini-ghosh) [reply frontend/src/pages/Dashboard/index.tsx:124]: done

File: frontend/src/components/PageHeader/types.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export interface IPageHeader {
```
[the remark this answers is not in the document]
Comment (nandini-ghosh) [reply frontend/src/components/PageHeader/types.ts:1]: done

**Review by kennybc — APPROVED** [review]
your button technically "breaks" some stuff im doing because you dont pass through the className so if i want to apply a custom class to my button it wont work, but no worries ill amend this in my PR soon

**Comment by nandini-ghosh** [comment]
@kennybc whoops sorry about that, lemme know if there's anything I can do to help

**Pull request merged**